packages/ckeditor5-link/src/linkediting.js

```diff
@@ -578,6 +587,29 @@ export default class LinkEditing extends Plugin {
 			} );
 		}, { priority: 'low' } );
 	}
+
+	/**
+	 * Enables autolinking on pasting.
+	 *
+	 * @private
+	 */
+	_enablePastingHandling() {
+		const editor = this.editor;
+		const model = editor.model;
+
+		this.listenTo( editor.plugins.get( 'ClipboardPipeline' ), 'contentInsertion', ( _, data ) => {
+			const defaultProtocol = this.editor.config.get( 'link.defaultProtocol' );
```

Comment:
`addLinkProtocolIfApplicable` will do nothing when `defaultProtocol` is undefined. However, I'd assume this is more common case and skip adding listener when there is no `'link.defaultProtocol'` config.